tests/CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ function(add_runtests targetname test_flags)
   )
 endfunction()
 
-add_runtests(test           "")
+add_runtests(test-all       "")
```

Comment:
Hmm, `test-all` sounds very similar to `test-full` to me 😕

Reply:
I am open to suggestions.
I have no idea what's the difference between "-a -p -r" and no-args. Do we even need the no-args version?

Reply:
@bagder any remarks on the name?

Reply:
I don't know. When are these target names used?

Those options are passed as-is to `runtests.pl` and they're documented here: https://curl.se/dev/runtests.html

Reply:
I don't think 'make test' is used anywhere in the CI. It looks to as the set of names and options is pretty arbitrary. It's clear we cannot use the simple name 'test', so the question is if anybody would miss it if we removed the target completely.